quicktest.go

```diff
@@ -164,22 +179,39 @@ func (c *C) Assert(got interface{}, checker Checker, args ...interface{}) bool {
 //             c.Assert(a, qt.Equals, 42)
 //         })
 //     }
-//
-// A panic is raised when Run is called and the embedded concrete type does not
-// implement Run, for instance if TB's concrete type is a benchmark.
 func (c *C) Run(name string, f func(c *C)) bool {
-	r, ok := c.TB.(interface {
-		Run(string, func(*testing.T)) bool
-	})
-	if !ok {
+	badType := func() {
 		panic(fmt.Sprintf("cannot execute Run with underlying concrete type %T", c.TB))
 	}
-	return r.Run(name, func(t *testing.T) {
-		c2 := New(t)
+	m := reflect.ValueOf(c.TB).MethodByName("Run")
+	if !m.IsValid() {
+		// c.TB doesn't implement a Run method.
```

Comment:
Why not including these comments (also the other cases below) in the panic message? That would speed up debugging in case of errors.

Reply:
done